### ISSUE TITLE
[Error Report] Crunchy [403: Forbidden] - JWT Error on Non-DRM Request Stream URLs

### ISSUE THREAD

**Issue by quamt** [anidl/multi-downloader-nx#697]
### Program version

5.0.0

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

NIL

### Show ID

ALL

### Episode

ALL

### Console Output

```Shell
[ERROR] 403: Forbidden
[ERROR] {"error":"JWT error"}
[ERROR] Non-DRM Request Stream URLs FAILED!
```

### Additional Information

The error appears for every stream, including when using DRM (DL not affected from DRM side), as AniDL looks for all streams/servers. 
It was reported elsewhere that Crunchy's side has made changes.

**Comment by Jaynator495** [comment]
This is already fixed, update to the latest version (5.0.1)

**Comment by quamt** [comment]
Thank you. Didnt see that.My aplologies.
